pkg/sql/opt/exec/factory.opt

```diff
@@ -620,6 +620,14 @@ define CreateView {
     withData bool
 }
 
+# CreateFunction implements CREATE FUNCTION.
```

Comment:
Ah...I just recall looking at the CI failure that looks like you'd need to move this to the bottom of the file to avoid the change in plan gist test?
I'm not sure are there any other impact other than the hashes changes in the test.

Reply:
Oh right, we have to add new operators to the end of this file. Thanks for pointing that out.